### DISCUSSION TITLE
Log4j vulnerability

### DISCUSSION THREAD

**Discussion by Coatsville** [requarks/wiki#4785] [Help / Questions] [answered]
I have been looking to see if there is any documentation that states whether Wiki.JS uses  log4j and, if so, is it using a version affected by the major vulnerability.

Can anyone tell me if Wiki.JS is affected by the Log4J/Log4Shell vulnerability?

**Comment by NGPixel** [comment, marked as answer]
Wiki.js is running on Node.js and is **NOT** affected by this vulnerability which is specific to Java applications.

**Reply by Coatsville** [reply]
Thanks Nicolas. I figured as much, but couldn't quite find documentation to confirm my suspicions. Thanks for the quick response!
